account_tax_balance/wizard/open_tax_balances.py

```diff
@@ -31,7 +31,25 @@ def onchange_date_range_id(self):
     def open_taxes(self):
         self.ensure_one()
         action = self.env.ref('account_tax_balance.action_tax_balances_tree')
-        vals = action.read()[0]
+        act_vals = action.read()[0]
+        # override action name doesn't work in v12 or v10
+        # we need to build a dynamic action on main keys
+        vals = {x: act_vals[x] for x in act_vals
```

Comment:
No need to make this filtering

Reply:
@pedrobaeza  If I do that it doesn't work, sorry ;-)

Reply:
That's very weird, as previous code was working also with a raw read...

Reply:
In previous code there was no override of the action name.

In returned vals the override name is transmit to all args of the stack but the problem comes from _compute_display_name() which compute another value than overrided name

Reply:
OK, let's go this way.

Reply:
Don't hesitate to try both code to see if I missed smth or not if any doubt 

Thanks Pedro

Reply:
And thanks for the explanation :)